client/my-sites/customer-home/locations/secondary/index.jsx

```diff
@@ -32,6 +32,7 @@ const Secondary = ( { cards } ) => {
 					cardComponents[ card ] &&
 					createElement( cardComponents[ card ], {
 						key: card,
+						siteId: siteId,
```

Comment:
In the future, we could redefine `cardComponents` above to include card specific props. But for now something like this should do it:

```suggestion
						...(card === SECTION_BLOGGING_PROMPT ? { siteId, showMenu: true, viewContext: 'home' } : {}),
```

Reply:
Thanks, added this.